client/post-editor/editor-title/index.jsx

```diff
@@ -4,6 +4,7 @@
 import React, { PropTypes } from 'react';
 import classNames from 'classnames';
 import omit from 'lodash/object/omit';
+import ReactDOM from 'react-dom';
```

Comment:
I may or may not have done this on purpose :trollface: - well it is what the deprecation notice told me to use in the console :laughing: